dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/cli/commands.py

```diff
@@ -93,6 +94,206 @@ def config(getter, setter, init) -> None:
         click.echo("Set configuration done.")
 
 
+@cli.command()
+@click.option(
+    "--set",
+    "-s",
+    "setter",
+    multiple=False,
+    type=click.Tuple([str, str, str, str, str, str, int]),
+    help="Set user to pydolphinscheduler."
+    "Use ``--set <NAME> <PASSWORD> <EMAIL> <PHONE> <TENANT> <QUEUE> <STATUS>`` options to set user",
```

Comment:
we currently use cli `user --set <NAME> <PASSWORD> <EMAIL> <PHONE> <TENANT> <QUEUE> <STATUS>`, but how about using `user create --user-name <NAME> --password <PASSWORD> --email <MAIL> ...` for more readable?

also for the update `user update <NAME> --email <MAIL> ...` for the part of update?

Reply:
Same as tenant and project